quodlibet/po/fr.po

```diff
@@ -4201,18 +4235,17 @@ msgstr ""
 msgid "Couldn't find GStreamer element '{element}'."
 msgstr ""
 
-#: ../quodlibet/plugins/playlist.py:31
+#: ../quodlibet/plugins/playlist.py:30
 #, python-format
 msgid "Run the plugin \"%(name)s\" on %(count)s playlist?"
 msgid_plural "Run the plugin \"%(name)s\" on %(count)s playlists?"
-msgstr[0] ""
-msgstr[1] ""
+msgstr[0] "Lancer le greffon \"%(name)s\" sur la liste %(count)s ?"
```

Comment:
"count" is the number of playlists the plugin will run on, so
-> "`Lancer le greffon \"%(name)s\" sur %(count)s liste de lecture`"